### PR TITLE
Track the worst frame in perf tests

### PR DESCRIPTION
**Pull request by Hixie**
@yjbanov

### CODE REVIEW

File: packages/flutter_driver/lib/src/timeline_summary.dart [hunk 1]
```diff
@@ -26,6 +27,8 @@ class TimelineSummary {
 
   /// Average amount of time spent per frame in the framework building widgets,
   /// updating layout, painting and compositing.
+  ///
+  /// Returns null if no frames were recorded.
```
Comment (yjbanov) [inline packages/flutter_driver/lib/src/timeline_summary.dart]: nit:

```
`null`
```

**Comment by yjbanov** [comment]
LGTM

**Comment by Hixie** [comment]
Fixed all cases of "/// Returns null if" to have backticks.

**Comment by yjbanov** [comment]
> Fixed all cases of "/// Returns null if" to have backticks.

Github is not showing it in the diff, but it was merely a nit, so still LGTM.

**Comment by Hixie** [comment]
I did something bad to this patch.

**Comment by Hixie** [comment]
Ok, it's fixed now. PTAL to make sure I am not crazy...

**Comment by yjbanov** [comment]
LGTM

**Pull request merged**